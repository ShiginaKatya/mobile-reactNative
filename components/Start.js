import { StatusBar } from 'expo-status-bar';
import { StyleSheet, Text, View, Image, Button } from 'react-native';

export default function Start({navigation}) {
  const loadScene = () => {
    navigation.navigate('Main')
  } 
  return (
    <View style={styles.container}>
      <Text style={styles.text}>BE BETTER</Text>
      <Image source={require('../assets/plant_2820715 1.png')} />
      <Text style={styles.text}>Привет!</Text>
      <Text style={styles.text_simple}>С BE BETTER ты можешь стать лучше вместе с нашей планетой!</Text>
      <Button color={'green'} title='Старт' onPress={loadScene} />
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
    color: 'green',
  },
  text: {
    color: 'green',
    textAlign: 'center',
    padding: 10,
    fontWeight: 'bold',
    fontSize: 20
  },
  text_simple: {
    color: 'green',
    textAlign: 'center',
    padding: 20,
  }
  
});