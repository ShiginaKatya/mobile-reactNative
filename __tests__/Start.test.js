import React from 'react';
import renderer from 'react-test-renderer';
import Start from '../components/Start';


test('renders correctly', () => {
  const tree = renderer.create(<Start />).toJSON();
  expect(tree).toMatchSnapshot();
});