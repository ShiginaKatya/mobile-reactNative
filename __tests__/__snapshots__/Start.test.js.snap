// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "color": "green",
      "flex": 1,
      "fontFamily": "Roboto",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "green",
        "fontSize": 20,
        "fontWeight": "bold",
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    BE BETTER
  </Text>
  <Image
    source={
      {
        "testUri": "../../../assets/plant_2820715 1.png",
      }
    }
  />
  <Text
    style={
      {
        "color": "green",
        "fontSize": 20,
        "fontWeight": "bold",
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    Привет!
  </Text>
  <Text
    style={
      {
        "color": "green",
        "padding": 20,
        "textAlign": "center",
      }
    }
  >
    С BE BETTER ты можешь стать лучше вместе с нашей планетой!
  </Text>
  <View
    accessibilityRole="button"
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "borderColor": "green",
        "borderRadius": 8,
        "borderWidth": 1,
        "justifyContent": "center",
        "marginBottom": 16,
        "marginHorizontal": "auto",
        "opacity": 1,
        "padding": 10,
        "width": "50%",
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#007aff",
            "fontSize": 17,
            "fontWeight": "normal",
            "textAlign": "left",
          },
          {
            "fontSize": 14,
            "textAlign": "center",
          },
          {
            "color": "green",
          },
          undefined,
        ]
      }
    >
      СТАРТ
    </Text>
  </View>
</View>
`;
